packages/app/components/stackbox/FrequencyOptionsCard.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+
+import { ChangeEvent, useEffect, useState } from "react";
+
+import { twMerge } from "tailwind-merge";
+
+import { FREQUENCY_OPTIONS } from "@/models";
+import { BodyText, RadioButton, TextInput } from "@/ui";
+
+interface FrequencyOptionsCardProps {
+  frequency: FREQUENCY_OPTIONS;
+  setEndDate: (date: Date) => void;
+}
+
+const defaultFrequencyOptions = {
+  [FREQUENCY_OPTIONS.hour]: ["12", "24", "72"],
+  [FREQUENCY_OPTIONS.day]: ["7", "15", "30"],
+  [FREQUENCY_OPTIONS.week]: ["5", "10", "20"],
+  [FREQUENCY_OPTIONS.month]: ["3", "6", "12"],
```

Comment:
here some suggestions:

```suggestion
  [FREQUENCY_OPTIONS.hour]: ["6", "12", "24"],
  [FREQUENCY_OPTIONS.day]: ["2", "5", "10"],
  [FREQUENCY_OPTIONS.week]: ["1", "2", "4"],
  [FREQUENCY_OPTIONS.month]: ["2", "6", "12"],
```
I use the day frequency, and usually is 3, 5 days. I think 7 15 30 are better reflected in weeks: 1,2,4.